Maddori.Apple/Maddori.Apple/Screen/MyReflection/Saved/MyReflectionDetailTableViewCell.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  MyReflectionDetailTableViewCell.swift
+//  Maddori.Apple
+//
+//  Created by 이성호 on 2022/11/06.
+//
+
+import UIKit
+
+import SnapKit
+
+final class MyReflectionDetailTableViewCell: BaseTableViewCell {
+    
+    // MARK: - property
+    let titleLabel: UILabel = {
```

Comment:
p5;

혹쉬,,, 한칸... 🙏
```suggestion
    // MARK: - property
    
    let titleLabel: UILabel = {
```

Reply:
[3899eac](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/76/commits/3899eac02bd788f7c7d29caae64cf58ab78e97fd)

넵넵